src/olympia/browse/templates/browse/macros.html

```diff
@@ -3,21 +3,21 @@
   <div class="other-categories highlight">
     <h3><a href="{{ base_url }}">{{ title }}</a></h3>
     {% if filter %}
-      <ul>
+     <ul>
         {% for key, title_ in filter.opts_dict.items() %}
           <li id="p-{{ key }}">
             <a href="{{ base_url|urlparams(sort=key) }}">{{ title_ }}</a>
           </li>
         {% endfor %}
-      </ul>
     {% endif %}
-    <ul>
+      <h3>{{ _('Categories') }}</h3>
```

Comment:
You can't stick a `h3` in the middle of a `ul` like this. You'll need to re-instate the closing an opening `ul`